sdks/python/apache_beam/examples/snippets/transforms/elementwise/mltransform_test.py

```diff
@@ -57,6 +57,7 @@ def check_mltransform_compute_and_apply_vocabulary_with_scalar():
 Row(x=array([4]))
 Row(x=array([1]))
 Row(x=array([0]))
+Row(x=array([0]))
```

Comment:
we also need to replicate this on the website, can be a separate PR.

Reply:
this is the same example that will be updated on the website with this change.

Reply:
ah, yes this is a snippet. thanks